### ISSUE TITLE
Loss 问题请教

### ISSUE THREAD

**Issue by Lzb68799** [JN-Yang/PConv-SDloss-Data#14]
您好作者，请问为什么我在替换了SD LOSS之后会一直出现张量在CPU和CUDA导致的超时错误呢，换回CIOU就能正常运行

**Comment by JN-Yang** [comment]
可以把这行(源码第108行)改为：

beta = torch.where(beta > delta, torch.tensor(delta, device=beta.device), beta)

**Comment by Lzb68799** [comment]
问题解决，太感谢了作者大大
